src/runtime.js

```diff
@@ -370,5 +361,5 @@ function variable_reachable(variable) {
 }
 
 function window_global(name) {
-  return window[name];
+  return globalThis[name];
```

Comment:
Adopting globalThis so that the runtime can be used more easily in non-windowed environments such as Node and Web Workers.